doc/release-notes-21056.md

```diff
@@ -0,0 +1,6 @@
+RPC client functionality
+------------------------
+
+- The new `-rpcwaittimeout` argument to `bitcoin-cli` sets the timeout
+  in seconds to use with `-rpcwait`. If the timeout expires
+  `bitcoin-cli` will report a failure. (#21056)
```

Comment:
Among the current headers in `doc/release-notes.md`, "New settings" might be the closest, but I'm not sure. There doesn't seem to be a CLI header. Suggestion follows, feel free to ignore.

```diff
-RPC client functionality
-------------------------
+New settings
+------------
 
-- The new `-rpcwaittimeout` argument to `bitcoin-cli` sets the timeout
-  in seconds to use with `-rpcwait`. If the timeout expires
+- A new `-rpcwaittimeout` argument to `bitcoin-cli` sets the timeout
+  in seconds to use with `-rpcwait`. If the timeout expires,
   `bitcoin-cli` will report a failure. (#21056)
```

Reply:
I don't know if that header is better. As a compromise "New bitcoin-cli settings" maybe?

I do think it's important to keep mentions of server and other tools's command line arguments separated. Then again, all of this can be organized when merging the fragments—it's not an automatic process.

Reply:
Yes, "New bitcoin-cli settings" seems better to me, too. Though as mentioned in https://github.com/bitcoin/bitcoin/pull/21056#discussion_r576917497, if we add the argument to the existing `-rpcwait=` (as a CLI setting intended for human manual use rather than by client software, ISTM it may be less subject to API constraints and would be less verbose than `-rpcwait -rpcwaittimeout=`), it could be "Updated bitcoin-cli settings".